protocol/app/module_accounts_test.go

```diff
@@ -29,6 +30,8 @@ func TestModuleAccountsToAddresses(t *testing.T) {
 		clobmoduletypes.InsuranceFundName:      "dydx1c7ptc87hkd54e3r7zjy92q29xkq7t79w64slrq",
 		rewardsmoduletypes.TreasuryAccountName: "dydx16wrau2x4tsg033xfrrdpae6kxfn9kyuerr5jjp",
 		rewardsmoduletypes.VesterAccountName:   "dydx1ltyc6y4skclzafvpznpt2qjwmfwgsndp458rmp",
+		vestmoduletypes.CommunityTreasury:      "dydx15ztc7xy42tn2ukkc0qjthkucw9ac63pgp70urn",
```

Comment:
To confirm, will this address change if the root `community_treasury` string changes? It's possible someone makes a mistake and changes the root string. We should be able to catch such issues

Reply:
address changes if root string changes!